src/content/docs/features/updater.mdx

```diff
@@ -3,12 +3,362 @@ title: Updater
+			[target."cfg(not(any(target_os = \"android\", target_os = \"ios\")))".dependencies]
+			tauri-plugin-updater = "2.0.0-beta"
+			# alternatively with Git:
+			tauri-plugin-updater = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v2" }
+			```
+
+		2. Modify `lib.rs` to initialize the plugin:
+
+			```rust title="src-tauri/src/lib.rs" ins={3-7}
+			fn run() {
+					tauri::Builder::default()
+							.setup(|app| {
+									#[cfg(desktop)]
+									app.handle().plugin(tauri_plugin_updater::Builder::new().build())?;
+									Ok(())
+							})
+							.run(tauri::generate_context!())
+							.expect("error while running tauri application");
+			}
+			```
+
+		3. Install the JavaScript Guest bindings using your preferred JavaScript package manager:
+
+			<CommandTabs
+				npm = "npm install @tauri-apps/plugin-updater"
+				yarn = "yarn add @tauri-apps/plugin-updater"
+				pnpm = "pnpm add @tauri-apps/plugin-updater"
+			/>
+		</Steps>
+	</TabItem>
+</Tabs>
+
+## Signing Updates
+
+Tauri's cli has a built-in signature mechanism to ensure that updates are safe to be installed.
+
+To sign your updates you need two things:
+
+1. The _public key_, which will be added to your `tauri.conf.json` file later, to validate update artifacts before the installation.
+2. The _private key_, which is used to sign your update artifacts and should NEVER be shared with anyone. Also, if you lose this key, you will NOT be able to publish new updates to your current user base. It is crucial to store it in a safe place where you can always access it.
+
+To generate the keys on Linux and macOS you can use the Tauri CLI:
+
+<CommandTabs
+	npm="npm run tauri signer generate -w ~/.tauri/myapp.key"
+	yarn="yarn run tauri signer generate -w ~/.tauri/myapp.key"
+	pnpm="pnpm tauri signer generate -w ~/.tauri/myapp.key"
+	cargo="cargo tauri signer generate -w ~/.tauri/myapp.key"
+/>
+
+If you are on Windows, you should use `$HOME/.tauri/myapp.key` or a different path of your choice instead:
+
+<CommandTabs
+	npm="npm run tauri signer generate -w $HOME/.tauri/myapp.key"
+	yarn="yarn run tauri signer generate -w $HOME/.tauri/myapp.key"
+	pnpm="pnpm tauri signer generate -w $HOME/.tauri/myapp.key"
+	cargo="cargo tauri signer generate -w $HOME/.tauri/myapp.key"
+/>
+
+## Tauri Configuration
+
+Now you need to configure Tauri's updater. To do this, add this to your Tauri config:
+
+```json title="tauri.conf.json"
+{
+	"plugins": {
+		"updater": {
+			"endpoints": [
+				"https://releases.myapp.com/{{target}}/{{arch}}/{{current_version}}"
+			],
+			"pubkey": "YOUR_UPDATER_SIGNATURE_PUBKEY_HERE",
+			"windows": {
+				"installMode": "passive",
+				"installerArgs": []
+			}
+		}
+	}
+}
+```
+
+The required keys are `"endpoints"` and `"pubkey"` to enable the updater.
+
+`"endpoints"`<a id="config-enpoints"/>:  must be an array of updater endpoint URLs as strings. TLS is enforced in production mode.<br/>
```

Comment:
```suggestion
`"endpoints"`<a id="config-enpoints"/>:  must be an array of updater endpoint URLs as strings. TLS is enforced in production mode.<br/>
All endpoints should generally return the same content, Tauri will fall back to the next endpoint URL if it couldn't connect to the first one or if it returned a non-2XX status code.<br/>
```
The logic in the plugin looks a bit off on first glance but this is how it worked in v1 and afaik how it's still supposed to work. Also adding this because of discord questions :)